docs/ONNX_Runtime_Perf_Tuning.md

```diff
@@ -1,39 +1,40 @@
 ## Why do we need to tune performance?
-ONNX Runtime is designed to be open and scalable, it created the concept of "Execution Provider" to represents different execution kernels. 
+ONNX Runtime is designed to be open and extensible with its concept of "Execution Provider" to represents different execution kernels. See the [design overview](./HighLevelDesign.md). 
 
-ONNX Runtime right now supports 4 CPU execution providers, which are, default(MLAS), MKL-ML, MKLDNN and nGraph. For nVidia GPU, we support CUDA and TensorRT execution providers. (Technically, MKL-ML is not an formal execution provider since it can only enabled by using build options and does not support GetCapability interface.)
-For different models and different hardware, there is no silver bullet which can always perform the best. And even for a single execution provider, many times you have several knobs to tune, like thread number, wait policy etc.
+ONNX Runtime supports a variety of execution providers across CPU and GPU: [see the list here](../README.md#high-performance).
+For different models and different hardware, there is no silver bullet which can always perform the best. Even for a single execution provider, often there are several knobs that can be tuned (e.g. thread number, wait policy etc.).
 
-This document will document some basic tools and knobs you could leverage to find the best performace for your model and your hardware.
+This document covers basic tools and knobs that can be leveraged to find the best performance for your model and hardware.
 
+## Is there a tool to help with performance tuning?
+Yes, the onnxruntime_perf_test.exe tool (available from the build drop) can be used to test various knobs. Please find the usage instructions using `onnxruntime_perf_test.exe -h`.
 
-## How do I use different execution providers?
-**Please be kindly noted that this is subject to change. We will try to make it consistent and easy to use across different language bindings in the future.**
+Additionally, the [ONNX Go Live "OLive" tool](https://github.com/microsoft/OLive) provides an easy-to-use pipeline for converting models to ONNX and optimizing performance with ONNX Runtime. The tool can help identify the optimal runtime configuration to get the best performance on the target hardware for the model.
+
+## Using different execution providers
 
 ### Python API
-For official python package which are released to Pypi, we only support the default CPU (MLAS) and default GPU execution provider. If you want to get other execution providers,
-you need to build from source.
+Official Python packages on Pypi only support the default CPU (MLAS) and default GPU (CUDA) execution providers. For other execution providers, you need to build from source. Please refer to the [build instructions](../BUILD.md). The recommended instructions build the wheel with debug info in parallel.
+
+For example: 
+
+`MKLDNN:		 ./build.sh --config RelWithDebInfo --use_mkldnn --build_wheel --parallel`
```

Comment:
Wondering if you can highlight the '--build_wheel' part. The '--build_wheel' flags is for enabling python.